src/screens/AdminPortal/OrgList/modal/OrganizationModal.tsx

```
@@ -128,12 +128,11 @@ const OrganizationModal: React.FC<InterfaceOrganizationModalProps> = ({
           data-testid="modalOrganizationDescription"
           autoComplete="off"
         />
-        <Form.Label>{tCommon('address')}</Form.Label>
+        <label>{tCommon('address')}</label>
```

Biome (2.3.14)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)